spec/controllers/attachments_controller_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe AttachmentsController, type: :controller do
+  let(:user) { create :user }
+  before { sign_in user }
+
+  describe 'Filters' do
+    it { should use_before_action(:authenticate_user!) }
+  end
+
+  describe 'GET #index' do
+    context 'when the user has at least one attachment' do
+      it 'redirect to the last attachment detail' do
+        create_list :attachment, 10, user_id: user.id
+        last_attachment = user.attachments.last
+
+        get :index
+
+        expect(response).to redirect_to results_attachment_path(last_attachment)
+      end
+    end
+
+    context 'not have any attachments' do
```

Comment:
How about

```suggestion
    context 'when the user does not have any attachments' do
```

to be consistent with the test above